site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -208,6 +208,51 @@ The number of days before a child account that has not yet been verified by a pa
 +
 Required when the [field]#Delete unverified children# toggle is enabled.
 
+==== Multi-Factor
+
+image::tenant-configuration-mfa-authenticator-settings.png[Tenant Configuration - MFA authenticator settings,width=1200,role=shadowed top-cropped]
+
+===== Authenticator settings
+
+[.api]
+[field]#Enabled# [optional]#Optional#::
+When enabled, users may use an authenticator application to complete a two-factor authentication request.
+
+image::tenant-configuration-mfa-email-settings.png[Tenant Configuration - MFA email settings,width=1200,role=shadowed top-cropped]
+
+===== Email settings
+
+[.api]
+[field]#Enabled# [optional]#Optional#::
+When enabled, users may use an email address to complete a two-factor authentication request.
+
+[field]#Template# [required]#Required#::
+The email template to use when a two-factor authentication request is sent to the User's email address.
++
+Required when the [field]#Enabled# toggle is enabled.
+
+image::tenant-configuration-mfa-sms-settings.png[Tenant Configuration - MFA SMS settings,width=1200,role=shadowed top-cropped]
+
+===== SMS settings
+
+[.api]
+[field]#Enabled# [optional]#Optional#::
+When enabled, users may use a mobile phone number to complete a two-factor authentication request.
+
+[field]#Messenger# [required]#Required#::
+The Messenger used to deliver the template.
+
+[field]#Template# [required]#Required#::
+The SMS template to use when a two-factor authentication request is sent to the User's mobile phone number.
++
+Required when the [field]#Enabled# toggle is enabled.
```

Comment:
This is inconsistent in the tenants core concepts. With the `email` section we do this, with the `family` section we don't. We should probably be consistent across the whole page. 

This kind of message  is more consistent with the API docs, so I'd lean toward it. but the API docs mark the field that is required if some other field is set as optional, not required. 

What do you think is the better way to do it?

Reply:
I could go either way. It looks like its *mostly* done on this page by listing the field as `Required` AND mentioning required when toggled.  I do like how we list as `Optional` in the api docs.  I left for now but happy to update if you feel strongly one way or another.

Reply:
I think we should be consistent within the page so let's do the 'required' and mentioning required when toggled.

Can you make the changes to the other parts of the page to make this consistent, please?